OctoKit/Issue.swift

```diff
@@ -8,6 +8,12 @@ public enum Openness: String {
     case Closed = "closed"
 }
 
+public enum State: String {
```

Comment:
Could you expand the `Openness` enum and all `All`? This way we use the same enum for describing the same purpose (state of an issue in model and request).